atd-vze/src/Components/GridTable.js

```diff
@@ -330,6 +330,19 @@ const GridTable = ({
   if (searchParameters["column"] && searchParameters["value"]) {
     // We will need to be careful which operator we will be using depending
     // on whether the column is an integer or a string, etc.
+
+    // Create an array of the searchable filters so we can loop over them and delete
+    // any old search parameters from the 'where' query before adding new search parameters.
+    const searchableColumns = Object.keys(query.config.columns).filter(
+      columnKey => {
```

Comment:
The script works, but in the debugger I see this warning message:

```
Line 337:  Expected to return a value at the end of arrow function
```

Maybe there is a way to refactor in a way that does not show the warning in the future?

Reply:
ohh good catch, i can refactor.